### PR TITLE
Device portal warnings

### PR DESCRIPTION
**Pull request by evmill**

### CODE REVIEW

**Comment by PRMerger10** [comment]
@evmill : Thanks for your contribution! The author(s) have been notified to review your proposed change. @hamalawi

File: mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-the-windows-device-portal.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ The Windows Device Portal for HoloLens lets you configure and manage your device
 
 This documentation is specifically about the Windows Device Portal for HoloLens. To use the Windows Device portal for desktop (including for Windows Mixed Reality), see [Windows Device Portal overview](/windows/uwp/debug-test-perf/device-portal)
 
+> [NOTE!]
+> Using device portal is not recommended to be used for HoloLens devices you are deploying to your organization.
```
Comment (pbarnettms) [inline mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-the-windows-device-portal.md:27]: This is not a correct/complete sentence.  Remove 'Using' and capitalize Device Portal.  I'd also like to see the statement be stronger -- "Device Portal is a developer tool only and is not to be used in applications deployed to your organization."

File: mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-the-windows-device-portal.md [hunk 2]
```diff
@@ -171,6 +174,9 @@ Once the installation is complete, go back to the **All apps** page on your Holo
 ![Windows Device Portal home page on Microsoft HoloLens](images/using-windows-portal-img-04.png)<br>
 *Windows Device Portal home page on Microsoft HoloLens*
 
+> [NOTE]
+> Settings configured in the device portal apply to the entire device and persist through reboots. Because of this it is recommended to only use device portal when developing, and not on deployed devices.
```
Comment (pbarnettms) [inline mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-the-windows-device-portal.md:178]: Capitalize "Device Portal" (applies everywhere)

Comment (pbarnettms) [inline mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-the-windows-device-portal.md:178]: Device Portal is for use by developers, not in deployed applications.

File: mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-the-windows-device-portal.md [hunk 3]
```diff
@@ -232,12 +238,14 @@ To import maps and anchors, select **Upload a map file** and **Upload an anchor
...
 > On HoloLens, it's also possible to import and export the spatial mapping data base. However, this doesn't work on non-HoloLens devices.  
 
-
 ### Mixed Reality Capture
 
 ![Mixed Reality Capture page in Windows Device Portal on Microsoft HoloLens](images/using-windows-portal-img-07.png)<br>
 *Mixed Reality Capture page in Windows Device Portal on Microsoft HoloLens*
 
+> [IMPORTANT]
+> Settings configured in the device portal apply to the entire device and persist through reboots. Any settings modified in the device portal will apply to mixed reality captures and apps. Because of this persistance, it is recommended to only use device portal when developing, and not on deployed devices.
```
Comment (pbarnettms) [inline mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-the-windows-device-portal.md:247]: Same as previous comment about capitalization and using a stronger statement.

**Review by pbarnettms — CHANGES_REQUESTED** [review]

**Comment by PRMerger15** [comment]
@evmill : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by vtieto** [comment]
Closing this PR. It was duplicated in PR #3324.

**Pull request closed without merging**